sources/engine/Stride.Graphics/OpenGL/GLCoreLibraryNameContainer.cs

```diff
@@ -12,22 +12,22 @@ namespace Silk.NET.OpenGL
     internal class GLCoreLibraryNameContainer : SearchPathContainer
     {
         /// <inheritdoc />
-        public override string Linux => "libGL.so.1";
+        public override string[] Linux => new[] { "libGL.so.1" };
```

Comment:
Can you make those properties `{ get; } =` instead of `=>` to avoid per call allocations of the array?

Reply:
I'll look into this as soon as I can.

Reply:
I would like to mention I modeled how I did it based on how Silk.NET internally does the same thing. e.g. https://github.com/dotnet/Silk.NET/blob/da7cae57614f91d0387fa51af8a52c99b7c43d98/src/Windowing/Silk.NET.SDL/SDLLibraryNameContainer.cs

Reply:
In fact, does it even make sense in the first place to have this file here, instead of just the one already in Silk.NET?